plugins/flutter_aepconsent/lib/flutter_aepconsent.dart

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import 'dart:async';
+import 'package:flutter/services.dart';
+
+/// Adobe Experience Platform Consent API.
+class Consent {
+  static const MethodChannel _channel = 
+       const MethodChannel('flutter_aepconsent');
+
+  /// Gets the current AEPConsent extension version.
+  static Future<String> get extensionVersion =>
+      _channel.invokeMethod('extensionVersion').then((value) => value!);
+
+  /// Retrieves the current consent preferences stored in the Consent extension
+  static Future<Map<dynamic, dynamic>> get consents => _channel
+      .invokeMethod<Map<dynamic, dynamic>>('getConsents')
+      .then((value) => value!);
+
+  /// Update the consent preferences stored in the Consent extension
```

Comment:
Would be nice to have more detailed API docs in this public file, similar to the react native one